source/config/configFlags.py

```diff
@@ -0,0 +1,37 @@
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2022 NV Access Limited, Cyrille Bougot
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+
+"""Flags used for the configuration.
+"""
+from enum import unique
+from utils.displayString import DisplayStringIntEnum
+
+
+@unique
+class ReportTableHeaders(DisplayStringIntEnum):
+	"""The possible config values to report table headers.
+	"""
+	
+	OFF = 0
+	ROWS_AND_COLUMNS = 1
+	ROWS = 2
+	COLUMNS = 3
+	
+	@property
+	def _displayStringLabels(self):
+		return {
+			# Translators: This is the label for a combobox in the
```

Comment:
The translator comment for these options seems wrong - these are not labels for the combo box.